mail/urlscan/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        firecat53 urlscan 0.9.5
+categories          mail python
+platforms           darwin
+license             GPL-2
+maintainers         {@ryanakca debian.org:rak}
```

Comment:
can `openmaintainer` be added here so that others can make minor changes without your permission?